Runtime/MuxySDK.cs

```diff
@@ -61,7 +66,7 @@ public UInt16 AuthenticateWithRefreshToken(string RefreshToken, AuthenticationCa
             });
 
             Handle = GCHandle.Alloc(WrapperCallback, GCHandleType.Pinned);
-            return Imported.AuthenticateWithRefreshToken(this.Instance, this.ClientId, RefreshToken, WrapperCallback, IntPtr.Zero);
+            return Imported.AuthenticateWithGameIDAndRefreshToken(this.Instance, this.ClientId, this.GameId, RefreshToken, WrapperCallback, IntPtr.Zero);
```

Comment:
Not all gamelink connections will have a gameID, this shouldn't assume that they will?